src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_inpaint.py

```diff
@@ -202,7 +202,8 @@ def __call__(
         self.scheduler.set_timesteps(num_inference_steps, **extra_set_kwargs)
 
         # preprocess image
-        init_image = preprocess_image(init_image).to(self.device)
+        if not isinstance(init_image, torch.FloatTensor):
+            init_image = preprocess_image(init_image)
```

Comment:
Why not put the `.to(self.device)` here, for symmetry with the mask and perhaps a bit more clarity? Currently it's a bit "hidden" a couple of lines below inside the call to `self.vae.encode()`.

Reply:
good point! Updated this to make device placement more clear. It's now after the if block for both mask and init image.